docs/guides/transfer-warp-route-ownership.mdx

```diff
@@ -85,7 +85,7 @@ hyperlane warp read --address <addressOrDenom> --chain yourchain
 
 After running `warp read`, you should see a similar config with the now updated `owner`:
 
-```yaml
+```yaml {3}
```

Comment:
![image](https://github.com/user-attachments/assets/36414ad5-37ed-4dca-84fe-8ded142decc8)
not sure i see a diff

Reply:
![Screen Shot 2024-08-06 at 12 35 51 PM](https://github.com/user-attachments/assets/d2b2324a-288a-4514-aa60-aad25cbedd30)
I think you're looking at the wrong area

Is there any way to make the highlight a little easier to read? on light mode the highlight is a bit aggressive

Reply:
ahhh nice

Reply:
I would fix the highlight intensity as a separate PR (or even separate Issue). This PR adds the highlight so that this part of documentation is consistent with other parts like `Step 3: Confirm` in 
https://docs.hyperlane.xyz/docs/guides/transfer-mailbox-ownership